src/main/webapp/app/pool/pool-booking/pool-booking.component.html

```diff
@@ -17,6 +17,18 @@ <h1 class="col-md-12" jhiTranslate="pools.booking.title">Book a pool</h1>
             Value should be greater than 0
         </small>
     </div>
+    <div>
+        <label jhiTranslate="pools.booking.fields.from">Count</label>
```

Comment:
At least make it required because i can make a reservation request without any data right now

Also: 
- amazing count as label in html
- What are those dates the input? Why we can't have inputs itself? And they are in weird format for a human being like: `2019-05-17T10:20>`

Reply:
fixed